README.md

```diff
@@ -918,11 +867,17 @@ gem:
   rbi_max_line_length: 120
   environment: development
   halt_upon_load_error: true
+  annotations: true
+  annotation_sources:
+  - https://raw.githubusercontent.com/Shopify/rbi-central/main
+  annotation_netrc: true
+  annotation_netrc_file: ''
+  annotation_auth: ''
```

Comment:
Let's pluralize all of them, we pull more than one annotation. This will allow be more cohesive with `annotations: true`